src/Entity/TurnkeyProject.php

```diff
@@ -175,6 +177,18 @@ class TurnkeyProject
      */
     private $position = 1;
 
+    /**
+     * @var Collection|TurnkeyProjectFile[]
+     *
+     * @ORM\ManyToMany(targetEntity="TurnkeyProjectFile", cascade={"persist"}, orphanRemoval=true)
+     * @ORM\JoinTable(name="turnkey_projects_has_files")
```

Comment:
why the default value `turnkey_projects_files` is not OK?
the `has` in table name is not really meaningful.